libs/partners/openai/langchain_openai/embeddings/base.py

```diff
@@ -282,12 +282,32 @@ def validate_environment(cls, values: Dict) -> Dict:
             "default_headers": values["default_headers"],
             "default_query": values["default_query"],
         }
+        
+        openai_proxy = values["openai_proxy"]
         if not values.get("client"):
+            if openai_proxy and not values["http_client"]:
```

Comment:
@baskaryan what pattern do we use here if openai_proxy and http_client have been co-specified? Do we usually raise or issue a warning?

Reply:
raise a warning if openai_proxy and either of http_client or http_async_client is specified

Reply:
@PabloKarpacho could we update to raise warning in above case ^

Reply:
Sure, do you have example in your code&